components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/js/JsAuthenticationContext.java

```diff
@@ -55,6 +55,8 @@ public Object getMember(String name) {
             return new JsAuthenticatedUser(getWrapped().getLastAuthenticatedUser());
         case FrameworkConstants.JSAttributes.JS_TENANT_DOMAIN:
             return getWrapped().getTenantDomain();
+         case FrameworkConstants.JSAttributes.JS_SERVICE_PROVIDER_NAME:
```

Comment:
Looks like some alignment mismatch.

Reply:
Fixed formatting.

Reply:
@mefarazath , is this fixed now? seems the issue is still here

Reply:
Yes. I have pushed the changes now.